scheduler/services/resource/ocs_resource_service.py

```diff
@@ -515,6 +532,86 @@ def none_to_str(value) -> str:
                     s = self._negative_filters[site].setdefault(d, set())
                     s.add(LgsFilter())
 
+    def _parse_eng_task_file(self, site: Site, to_file: str) -> None:
+        """
+        Parse Engineering task that block moments or the entire night.
+        Each twilight is calculated using lucupy.sky, some discrepancies might affect.
+        """
+        pattern = r'(\[.*\])'
+        # Ignore GS until the file is created.
+        if site is Site.GS:
+            return
+
+        with open(os.path.join(self._path, to_file), 'r') as file:
+            for line in file:
+                # Find pattern to keep bracket comments not splitted
+                match = re.search(pattern, line)
+                if match:
+                    comment = match.group(1)
+                    rest_of_line = re.sub(pattern, '', line)
+                    _date, start_time, end_time = rest_of_line.strip().split()
+                    #  Single date for key
+                    just_date = datetime.strptime(_date, '%Y-%m-%d')
+                    # Time day in jd to calculate twilights
+                    time = Time(just_date)
+                    _, _, _, even_12twi, morn_12twi, _, _ = night_events(time,
```

Comment:
This is kind of fragile in case we ever change the structure of `NightEvent` or the ordering of its elements.

Maybe we should do:
```python
ne = night_events(time, site.location, site.timezone)
eve_12twi = ne.twilight_evening_12
mor_12twi = ne.twilight_morning_12
```

Reply:
I agree. I change those.